src/actions/generateDefaultMetadata.ts

```diff
@@ -0,0 +1,22 @@
+import { DomainMetadata } from "../types";
+import { ApiClient } from "../api";
+import { getLogger } from "../utilities";
+
+const logger = getLogger("actions:generateDefaultMetadata");
+
+export const generateDefaultMetadata = async (
+  apiClient: ApiClient,
+  name: string
+): Promise<string> => {
+    const domainMetaData: DomainMetadata = {
+        name: `0://${name}`,
+        description: `0://${name} - A Zero Name Service (zNS) Root Domain on the Ethereum Blockchain`,
+        image: "ipfs://QmS6DZzG6pTdWszubmDXzJ7vcEWsPNMGJZuviDaCMc7beB",
```

Comment:
Consider making these named constants for clarity.

`image: constants.DEFAULT_IMAGE` or similar